Formula/e/ehco.rb

```diff
@@ -22,22 +23,21 @@ class Ehco < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "566974576744f5056e4a7d4f5d3b14b0b6a4af3f25a8751f5fa5c1cf901c6e83"
   end
 
-  depends_on "go@1.19" => :build
+  depends_on "go@1.20" => :build
 
   uses_from_macos "netcat" => :test
 
-  # Backport quic-go update to support Go 1.19
-  # Remove in the next release
+  # go@1.20 build patch, remove in next release
```

Comment:
If this is already fixed upstream, can we move it to a `stable` block?

Reply:
yeah, that sounds about right